draft-ietf-tls-sni-encryption.md

```diff
@@ -57,64 +56,66 @@ and that practical solutions will have to make some compromises.
 # Introduction
 
 Historically, adversaries have been able to monitor the use of web
-services through three channels: looking at DNS requests, looking at
-IP addresses in packet headers, and looking at the data stream between
+services through three primary channels: looking at DNS requests, looking 
+at IP addresses in packet headers, and looking at the data stream between
 user and services. These channels are getting progressively closed.
 A growing fraction of
 Internet communication is encrypted, mostly using Transport Layer Security
 (TLS) [@?RFC5246].
 Progressive deployment of solutions like DNS in
-TLS [@?RFC7858] mitigates the disclosure of DNS information. More and
+TLS [@?RFC7858] and DNS over HTTPS [@?RFC8484] 
+mitigates the disclosure of DNS information. More and
 more services are colocated on multiplexed servers, loosening the
 relation between IP address and web service. However, multiplexed servers
-rely on the Service Name Information (SNI) to direct TLS connections
+rely on the Service Name Information (SNI) TLS extension to direct connections
 to the appropriate service implementation. This protocol element
 is transmitted in clear text. As the other methods of monitoring get
 blocked, monitoring focuses on the clear text SNI. The purpose
-of SNI encryption is to prevent that.
+of SNI encryption and privacy is to prevent that.
 
 In the past, there have been multiple attempts at defining SNI encryption.
 These attempts have generally floundered, because the simple designs fail
 to mitigate several of the attacks listed in (#snisecreq). In the absence of
-a TLS level solution, the most popular approach to SNI privacy is HTTP level
-fronting, which we discuss in (#httpfronting).
+a TLS-level solution, the most popular approach to SNI privacy for web
+services is HTTP-level fronting, which we discuss in (#httpfronting).
 
 # History of the TLS SNI extension
 
-The SNI extension was standardized in 2003 in [@?RFC3546] to 
-facilitate management
-of "colocation servers", in which a multiple services shared the same IP
+The SNI extension was specified in 2003 in [@?RFC3546] to facilitate management
+of "colocation servers", in which multiple services shared the same IP
 address. A typical example would be mutiple web sites served by the
 same web server. The SNI extension carries the name of a specific
 server, enabling the TLS connection to be established with the desired
 server context. The current SNI extension specification can be
 found in [@?RFC6066].
 
 The SNI specification allowed for different types of server names,
-but only the "hostname" variant was standardized and deployed. In that
+though only the "hostname" variant was standardized and deployed. In that
```

Comment:
to be consistent replace 'standardized' with 'defined' or 'specified'

Reply:
Also a good catch. Will fix.